zstacklib/zstacklib/test/test_safe_delete.py

```
@@ -0,0 +1,311 @@
+_LINUX_PY = os.path.join(_THIS_DIR, '..', 'utils', 'linux.py')
+
+import re as _re          # noqa: E402
+import logging as _logging  # noqa: E402
+
+_ns = {
+    'os': os,
+    're': _re,
+    'shutil': shutil,
+    '__builtins__': __builtins__,
+    'logger': _logging.getLogger('test_safe_delete'),
+    '_string_types': (str, unicode),
```

Ruff (0.15.9)

[error] Undefined name `unicode`

(F821)